GLTFSerialization/GLTFSerialization/Exceptions.cs

```diff
@@ -25,4 +25,16 @@ public class GLTFParseException : Exception
 		{ }
 #endif
 	}
+
+    public class GLTFLoadException : Exception
```

Comment:
>     [](start = 0, length = 4)

nit: should be tabs.  I know we're inconsistent in some files, but the .editorconfig says tabs.  So everything should be tabs.